config/initializers/gitlab_shell_secret_token.rb

```diff
@@ -14,6 +13,8 @@
   File.write(secret_file, token)
 end
 
-if File.exist?(Gitlab.config.gitlab_shell.path) && !File.exist?(gitlab_shell_symlink)
-  FileUtils.symlink(secret_file, gitlab_shell_symlink)
+# TODO May be removed after updating the default secret_file path...?
```

Comment:
Why? It doesn’t matter, I’ve already removed this note. Look, today it’s exactly 3 months since I’ve opened this PR. I want it to be finally merged.